### DISCUSSION TITLE
Deprecation messages are stored in the devtools-frontend repo at front_end/models/issues_manager/DeprecationIssue.ts.

### DISCUSSION THREAD

**Discussion by ezur1** [nuxt-community/sentry-module#446] [Q&A]
getting this error - 
**Deprecation messages are stored in the devtools-frontend repo at front_end/models/issues_manager/DeprecationIssue.ts.**
not sure how to reach the deprecation messages.

please help

**Comment by rchl** [comment]
Seems like this is Chrome message and not related to this module at all.

**Comment by AnthonyRuelle** [comment]
Hi, 

I've had the same problem with my project for a few weeks and it's getting annoying.
Why does Sentry raise this error? What does that mean ?

Could it be related to a too old version that could be solved by this PR?
https://github.com/nuxt-community/sentry-module/pull/425

**Comment by AnthonyRuelle** [comment]
Up, 

it's really very annoying, we are constantly polluted

**Reply by rchl** [reply]
As I wrote, this seems like a message triggered by chrome. At least it seems so after short googling.

Maybe it's sentry that indirectly triggers those but I can't tell since I'm not seeing those myself.

**Comment by GregorSondermeier** [comment]
Hello,

we got a Spike Warning from sentry because this error pollutes our issue log. Our website uses Nuxt 2. Is there any advice on how to avoid this message being logged?

**Reply by rchl** [reply]
Check sentry documentation for how to ignore specific errors

**Reply by GregorSondermeier** [reply]
Our solution:

In our _nuxt.config.js_ we have set the following:

```
  ...
  sentry: {
    ...
    clientIntegrations: {
      ReportingObserver: false,
    },
  },
  ...
```

Documentation:
https://sentry.nuxtjs.org/sentry/options#clientintegrations

We are aware that this doesn't fix the root issue, but at least it prevents these warnings from polluting our Sentry log :)

**Reply by rchl** [reply]
I'm disabling those noisy reports in the latest version: https://github.com/nuxt-community/sentry-module/pull/560
